### PR TITLE
Add starter issue ISSUE_TEMPLATE.md for GitHub

### PR DESCRIPTION
**Pull request by courajs**
@gr2m how's this look?
@renrutnnej Let me know if I let anything too coder/dev slip in there that should be changed for editorial issues

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+This is a [starter issue](http://hood.ie/blog/starter-issues.html).
+
+<!-- This section is optional. Some may have contributed before, but
+would like to do some more starter issue work to get more comfortable.
+You should aim to be very explicit for a Your First PR, but you can
+leave more up to the contributor if you don't reserve for first-timers -->
```
Comment (courajs) [inline ISSUE_TEMPLATE.md]: Especially looking for feedback on this ^

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+### ❓ The Motivation
+Bugs are pretty self-explanatory, but some features or changes might
+be less obvious. A successful PR is much more satisfying if you
+understand how it moves the project forward.
+
+<!-- Remove for feature work, editorial, etc. -->
+### 🐞 The Bug
+Steps to reproduce, or a snippet of sample code showing the bad behvior
+
+### :dart: The goal
```
Comment (courajs) [inline ISSUE_TEMPLATE.md:26]: And looking for feedback on `The goal` section

**Comment by jennwrites** [comment]
LGTM!
Thanks @courajs 🎉

**Comment by HipsterBrown** [comment]
LGTM! Great work @courajs! :clap:

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (gr2m) [inline ISSUE_TEMPLATE.md]: Maybe rephrase a bit, to something like this?

```
<!-- This section is optional. "Your First PR" issue are reserved for new contributors. We try to have issues available for both new and existing contributors at all times. If you leave the section in, add the "Your First PR" label -->
```

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+This should help a new contributor envision success as they get started,
+and answer the questions:
+- Am I done yet?
+- Did I forget anything?
+
+### :clipboard: Step by Step
+- [ ] claim this issue:
```
Comment (gr2m) [inline ISSUE_TEMPLATE.md]: add

> comment below. Once claimed we add you as contributor to this repository, so we can assign you to the ticket and you can start checking off the checkboxes below as you progress. We will also replace the up for grabs label with in progress.

Add two spaces after `claim this issue:` to force a line break

**Comment by courajs** [comment]
@gr2m better?

**Comment by gr2m** [comment]
purrfet

**Comment by gr2m** [comment]
purrfect
